evosax/strategies/lm_ma_es.py

```diff
@@ -171,87 +151,86 @@ def tell_strategy(
         # Sort new results, extract elite, store best performer
         concat_p_f = jnp.hstack([jnp.expand_dims(fitness, 1), x])
         sorted_solutions = concat_p_f[concat_p_f[:, 0].argsort()]
+        concat_z_f = jnp.hstack([jnp.expand_dims(fitness, 1), state.z])
+        sorted_zvectors = concat_z_f[concat_z_f[:, 0].argsort()]
+        sorted_z = sorted_zvectors[:, 1:]
+        if state.best_fitness < sorted_solutions[0, 0]:
+            state.best_fitness = sorted_solutions[0, 0]
+            state.best_member = sorted_solutions[0, 1:]
```

Comment:
Can you rewrite this with `jax.lax.select` -- otherwise JAX will have to recompile for the different cases when jitting.

Reply:
thanks for your kindness! I rewrite this code in the latest commit (and fixed the condition in minimize problem's way).

Reply:
Sorry, I found that ```Strategy.tell``` function updates them properly so these code are unnecessary; in the latest commit it is removed.